QuizApi/ServiceExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.Extensions.DependencyInjection;
+using QuizApp.MapperProfiles;
+using QuizApp.DAL.Repository;
+using QuizApp.DAL.Repository.Contracts;
+using QuizApp.Services.Contracts;
+using QuizApp.Services;
+
+namespace QuizApp
+{
+	public static class ServiceExtensions
```

Comment:
I would split this class into three different: one for repositories (RepositoryExtension), one for the mapper, and one for services.